.travis.yml

```diff
@@ -7,6 +7,9 @@ node_js:
 before_install:
   - sudo apt-get update
   - sudo apt-get install -y libsecret-1-dev
+script:
```

Comment:
Anuj, do we want the script to execute all the commands even if the first one fails?

Reply:
The commands are sequential, and next one is triggered only when the first one exits successfully. 
We'll need to trigger tests for ci to generate `lcov.info` file post which sonar can render coverage reports.

Reply:
Right, the Travis build would fail even if one cmd fails, I guess the commands would still execute even if the former ones fail..
I gathered this from [here](https://docs.travis-ci.com/user/job-lifecycle/#customizing-the-build-phase),
If the "run test" fails, sonar scan would still generate a report.
Not sure though, please have a look at the url.

Reply:
Thanks! 
Let's think about a case when `npm run test` fails due to unmet coverage threshold of 90% for nyc. Although the command failed, but coverage report ( `lcov.info` ) would be generated, which sonar can pick up to render % info about the coverage, although the travis build status is still `failed`. 
I hope it adds some details. Please let me know in case you think otherwise, as skipping this behaviour should be straight forward via chaining them up - `npm run test && sonar-scanner` (But this'd not show coverage reports in the above case!)

Reply:
I guess it makes sense to have the report generated every time we trigger a build. :)